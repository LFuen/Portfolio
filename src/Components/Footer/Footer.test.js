import React, { createElement } from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Footer from './Footer'


describe(`Footer component`, () => {
    describe(`Snapshot`, () => {
        it(`renders the UI as expected`, () => {
            const footer = renderer.create(
            <BrowserRouter>
                <Footer />
            </BrowserRouter>).toJSON()
            expect(footer).toMatchSnapshot()
        })
    })

    describe(`Smoke test`, () => {
        it(`renders without crashing`, () => {
            const div = document.createElement('div')

            ReactDOM.render(
                <BrowserRouter>
                    <Footer />
                </BrowserRouter>,
                div
            )
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})