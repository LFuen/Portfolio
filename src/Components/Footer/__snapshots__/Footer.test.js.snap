// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Snapshot renders the UI as expected 1`] = `
<footer
  className="foot"
>
  <p>
     Â© Lili Fuentes | 2020 
  </p>
</footer>
`;
